packages/sonarwhal/src/lib/utils/misc.ts

```diff
@@ -145,6 +145,10 @@ const normalizeStringByDelimiter = (value: string, delimiter: string) => {
     return normalizeString(value).replace(/[^a-z0-9]/gi, delimiter);
 };
 
+const isParentString = (str1: string, str2: string) => {
```

Comment:
Maybe something like `isNormalizedIncluded` ? Also I'd change `str1` for `source`, and `str2` for `included` (or maybe something different?).

And then add a description like:

> Checks if `source` contains `included` after normalizing both